OpenRA.Game/Widgets/Widget.cs

```diff
@@ -503,8 +503,11 @@ public virtual void HideChild(Widget child)
 
 		public virtual void RemoveChildren()
 		{
-			while (Children.Count > 0)
-				RemoveChild(Children[Children.Count - 1]);
+			foreach (var child in Children)
```

Comment:
Are we sure the order is not important here? I assume there is/was a reason for iterating over Children backwards before.

Reply:
Well gj me making two mistakes removing two lines. I'm not sure about the order either way (although I doubt it's important), but I'll reverse the `foreach` for good measure.

Reply:
Reversed.